### ISSUE TITLE
用两个机器，启动的时候，报错

### ISSUE THREAD

**Issue by lonelydancer** [TsinghuaAI/CPM-2-Finetune#8]
我看执行的如下的命令，
cmd = pdsh -f 1024 -w 192.168.10.25,192.168.10.24 export NCCL_INCLUDE_DIR=/usr/include; export NCCL_VERSION=2.7.8; export NCCL_SHM_DISABLE=1; export NCCL_DEBUG_SUBSYS=ALL; export NCCL_LIBRARY=/usr/lib/x86_64-linux-gnu; export NCCL_DEBUG=WARN; export NCCL_SOCKET_IFNAME=lo; export PYTHONPATH=/CPM/CPM-2-Finetune;  cd /CPM/CPM-2-Finetune; /opt/conda/bin/python -u -m deepspeed.launcher.launch --world_info=eyIxOTIuMTY4LjEwLjI1IjogWzAsIDEsIDIsIDNdLCAiMTkyLjE2OC4xMC4yNCI6IFswLCAxLCAyLCAzXX0= --node_rank=%n --master_addr=192.168.10.25 --master_port=29500 /CPM/CPM-2-Finetune//finetune_cpm2.py --model-config '/CPM/CPM-2-Finetune//configs/model/cpm2_config.json' --model_parallel_size '4' --batch-size '4' --gradient-accumulation-steps '1' --train-iters -1 --save '/CPM/CPM-2-Finetune//results/math/cpm2_finetune_lr0.000005const_G1/' --log-file '/CPM/CPM-2-Finetune//results/math/cpm2_finetune_lr0.000005const_G1//log.txt' --load '/CPM/cpm_model' --data-path '/CPM/CPM-2-Finetune/data/' --data-ext '.json' --data-name 'math' --distributed-backend 'nccl' --lr '0.000005' --no-load-optim --lr-decay-style 'constant' --weight-decay '1e-2' --clip-grad '1.0' --warmup '0.0' --tokenizer-path '/CPM/CPM-2-Finetune//bpe_cn' --save-interval '100000' --eval-interval '50' --eval-iters '10' --log-interval '10' --checkpoint-activations --deepspeed-activation-checkpointing --deepspeed --do-train --deepspeed_config '/CPM/CPM-2-Finetune//configs/deepspeed/ds_full_model.json' --do-valid --epochs '20'

报错信息是
192.168.10.25: Warning: Permanently added '192.168.10.25' (ECDSA) to the list of known hosts.
192.168.10.24: Warning: Permanently added '192.168.10.24' (ECDSA) to the list of known hosts.
192.168.10.25: Permission denied, please try again.
192.168.10.25: Permission denied, please try again.
192.168.10.25: root@192.168.10.25: Permission denied (publickey,password).
pdsh@8ad3b56dace0: 192.168.10.25: ssh exited with exit code 255
192.168.10.24: bash: line 0: cd: /CPM/CPM-2-Finetune: No such file or directory
192.168.10.24: bash: /opt/conda/bin/python: No such file or directory
pdsh@8ad3b56dace0: 192.168.10.24: ssh exited with exit code 127
是否在cmd中要进入一下docker环境?

**Comment by t1101675** [comment]
看 log 是您需要配置一下机器之间 ssh 的 publickey 验证？

**Comment by lonelydancer** [comment]
@t1101675 嗯。这里面有好几个问题。但是这个报错
192.168.10.24: bash: line 0: cd: /CPM/CPM-2-Finetune: No such file or directory
进入的是docker的路径。

**Comment by t1101675** [comment]
您可能需要在进入 docker 环境的时候将本目录挂载到 docker 的 /CPM/CPM-2-Finetune 目录下
